src/main/java/com/google/error_analyzer/backend/FileAndUrlLogs.java

```diff
@@ -41,6 +41,7 @@ public String checkAndStoreFileAndUrlLog(HttpServletRequest request,
             indexName = storeLogs.getUniqueIndexName(indexName);
             final String response = storeFileAndUrlLogs(
                 request, indexName, fileContent, isUrl);
+            storeLogs.findErrorsInIndex(indexName);
```

Comment:
I guess storeFileAndUrlLogs internally calls storeLogs.checkAndStoreLog with offset. This is not the right place to call it.
Call it in checkAndStoreLog with offset

Reply:
That would call the same function every time after storing 10k lines

Reply:
That would call the finAndStoreErrors function multiple times